tools/data/anno_txt2json.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+from mmcv import dump
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Convert txt anno list to json')
+    parser.add_argument(
+        'annofile', type=str, help='the txt annofile to convert')
```

Comment:
`annofile` -> `annotation file`

Reply:
'annofile' in comments is replaced as 'annotation file', while the arg name is not changed to keep it short